app/javascript/dashboard/components/widgets/conversation/MessagesView.vue

```diff
@@ -416,7 +416,7 @@ export default {
 
     .reply-box .reply-box__top {
       position: relative;
-      min-height: 44rem;
+      height: 50vh;
```

Comment:
Are we sure about this change? @iamsivin just verify this does not break on multiple screen sizes

Reply:
@scmmishra Here is the quick look related to this changes https://www.loom.com/share/9eaf3818f47c4e6dba267c02b861e6c7